lib/backends/webgl/ops/matmul-pack.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import {MatMul} from '../../../ops/matmul';
+import {Tensor} from '../../../tensor';
+import {BroadcastUtil} from '../../../util';
+import {WebGLInferenceHandler} from '../inference-handler';
+import {ProgramInfo, RunData, WebGLOperator} from '../types';
+
+export class WebGLMatMulPacked extends MatMul implements WebGLOperator {
+  run(inferenceHandler: WebGLInferenceHandler, inputs: Tensor[]): Tensor[] {
+    return inferenceHandler.run(this, inputs);
+  }
+  createProgramInfo(handler: WebGLInferenceHandler, inputs: Tensor[]): ProgramInfo {
+    const hasBias = inputs.length > 2;
+    const processBias = hasBias ? `value += vec4(getBias(a[0]*2).xx, getBias(a[0]*2).yy);` : ``;
+    const aShape = inputs[0].dims;
+    const bShape = inputs[1].dims;
+    const outputShape = BroadcastUtil.calcShape(aShape, bShape, true);
```

Comment:
I think our current implementation doesn't cover cases when one of the input tensor needs broadcast? If so, we may want to add a TODO here for future re-visiting.

Reply:
done